components/wpt-results.html

```diff
@@ -180,23 +180,6 @@
         this._refreshDisplayedSpecs()
       }
 
-      _specFilter(spec) {
-        if (this.query.length === 0) {
-          return true
-        }
-
-        const matchesQuery = (tf) => tf.testFile.toLowerCase().includes(this.query.toLowerCase())
-        return spec.testFiles.some(matchesQuery)
-      }
-
-      _testFileFilter(testFile) {
-        if (this.query.length < 3) {
-          return false
-        }
-
-        return testFile.testFile.toLowerCase().includes(this.query.toLowerCase())
-      }
-
       _testFileSort(a, b) {
         if (a.testFile < b.testFile) return -1;
         if (a.testFile > b.testFil) return 1;
```

Comment:
There's a typo here which I think could lead to incorrect sorting behavior if there were a list with some duplicate test file names. This line isn't changed in this PR, but could be fixed here.